### DISCUSSION TITLE
How do I login to Canva?

### DISCUSSION THREAD

**Discussion by JermT** [community/community#35323] [GitHub Education]
Hey everyone! 

I previously claimed the Canva Pro on Github before it was removed. I understand that the 12-month offer still stands.
I'm now struggling to login, how do I access Canva Pro with my github account?

Thank you!

**Comment by SilasRodrigues19** [comment]
@JermT Looks like Canva Pro is no longer available in the GitHub Developer Pack, it no longer appears on my list of benefits when I access the link below.

[https://education.github.com/pack/offers](https://education.github.com/pack/offers)

**Comment by angadisumanth10** [comment]
Canva Pro was Good earlier. So sad to see that it's not in the list :(

**Comment by itziklugassy** [comment]
there is no extension for that?
